tests/test_cli.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from click.testing import CliRunner
+from getgist.__main__ import (
+    run_getgist,
+    run_getmy,
+    run_putgist,
+    run_putmy,
+    run_lsgists,
+    run_mygists,
+)
+
+runner = CliRunner()
+
+
+@pytest.mark.parametrize(
+    "command, to_patch, args",
+    [
+        (run_getmy, None, []),
+        (run_putmy, "getgist.__main__.GetGist.put", []),
+        (run_putgist, None, ["cuducos"]),
+        (run_getgist, None, ["cuducos"]),
+    ],
+)
+def test_run_command_with_error_in_filename(
+    command, to_patch, args, mocker, monkeypatch
+):
+    monkeypatch.setenv("GETGIST_USER", "cuducos")
+
+    if to_patch:
+        mocker.patch(to_patch)
+
+    result = runner.invoke(command, args)
+    assert result.exit_code == 2
+    assert "Error: Missing argument 'FILENAME'." in result.output
+
+
+@pytest.mark.parametrize(
+    "command, to_patch, args",
+    [
+        (run_putgist, "getgist.__main__.GetGist.put", []),
+        (run_getgist, None, []),
+        (run_lsgists, "getgist.__main__.GetGist.put", []),
```

Comment:
What about `run_getmy`, `run_putmy` and `run_mygists` with no `GETGIST_USER` set? Can we have them too? If it's too much, we can leave for another issue/PR.